crypto/keys/private.go

```diff
@@ -56,9 +56,6 @@ type SignerFactory interface {
 // NewFromPrivatePEMFile reads a PEM-encoded private key from a file.
 // The key must be protected by a password.
 func NewFromPrivatePEMFile(keyFile, keyPassword string) (crypto.Signer, error) {
-	if keyPassword == "" {
```

Comment:
Please don't remove this check, we've added it on purpose. The intention is to foster good practices along with Trillian usage. People really shouldn't be storing unencrypted PEM files in their servers.

There's some context at #544. If you feel strongly about this, please move the discussion to a separate PR.